src/test/suite/fixtures/recorded/languages/python/changeCallee2.yml

```diff
@@ -0,0 +1,23 @@
+languageId: python
+command:
+  version: 1
+  spokenForm: change callee
+  action: clearAndSetSelection
+  targets:
+    - type: primitive
+      modifier: {type: containingScope, scopeType: functionCallee, includeSiblings: false}
+initialState:
+  documentContents: "object.method()\r\n"
```

Comment:
Not sure it's worth re-recording the test, but if you can avoid the `\r\n` by either using unix line endings (`\n`) or just not having new lines at all, the test case will be easier to read as yaml will be able to leave out the quotes

Reply:
I went ahead and rerecorded it.